### PR TITLE
2417 remove specialist topic code from publishing api l 4

### PR DESCRIPTION
**Pull request by unoduetre**
⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

This application is owned by the publishing platform team. Please let us know in #govuk-publishing-platform when you raise any PRs.

Follow [these steps](https://guides.rubyonrails.org/upgrading_ruby_on_rails.html) if you are doing a Rails upgrade.

Part of

[🏔Epic | Remove specialist topic code from Publishing API [L]EPICS 🌀](https://trello.com/c/2LcbNRSQ/2417-%F0%9F%8F%94epic-remove-specialist-topic-code-from-publishing-api-l "‌")

## What

Remove topics from email\_alert\_signup schema

## Why

Retiring specialist topics
[Trello card](https://trello.com/c/lJFANf8k/2488-remove-topics-from-emailalertsignup-schema-m-l)

## How

[https://github.com/alphagov/publishing-api/blob/main/docs/content_schemas/changing-an-existing-content-schema.md#removing-a-field-from-a-content-schema](https://github.com/alphagov/publishing-api/blob/main/docs/content_schemas/changing-an-existing-content-schema.md#removing-a-field-from-a-content-schema "‌")

### CODE REVIEW

**Comment by hannako** [comment]
I did some digging into where this schema is being used:
- It is used by one of the three email signup routes supported by [email alert frontend](https://github.com/alphagov/email-alert-frontend/blob/main/README.md?plain=1#L9), ie signups from `<base_path>/email-signup`, [see controller](https://github.com/alphagov/email-alert-frontend/blob/bea75b7d43519bc97c108fb2a2d36dab1c28be24/config/routes.rb#L6)
- this route is currently only used by travel advice pages eg https://www.gov.uk/api/content/foreign-travel-advice/france/email-signup
- the concept of the `email-alert-type` is I think legacy code from when signups were possible from policy pages (clues [here](https://github.com/alphagov/email-alert-frontend/commit/e89bb9a0efbacfc3582bda27f10facc0d2f425f1?diff=split&w=0), [here](https://github.com/alphagov/email-alert-frontend/pull/17) and [here](https://github.com/alphagov/email-alert-api/pull/93) ) so _probably_ the whole key could be removed. As it seems like on travel advice pages, the links contain the `countries` key.
- Removing the whole key is out of scope of this work, but just noting here to make the review easier:

**TLDR**: I don't think the `topic` field was being used anywhere anyway so there are no consequences of removing it.

**Review by hannako — APPROVED** [review]
I did some digging into where this schema is currently being used.
- It's used on travel advice sign up pages only. eg https://www.gov.uk/api/content/foreign-travel-advice/afghanistan/email-signup
- It doesn't look like the email_alert_type field is even used tho, as the countries key is in top level links
- I think the `policies` key (and potentially the whole `email_alert_type`) key could be removed too (clues [here](https://github.com/alphagov/email-alert-frontend/commit/e89bb9a0efbacfc3582bda27f10facc0d2f425f1?diff=split&w=0) [here](https://github.com/alphagov/email-alert-frontend/pull/17) and [here](https://github.com/alphagov/email-alert-api/pull/93)) but that's out of scope of this work. https://gov-uk.atlassian.net/wiki/spaces/GOVUK/pages/116490261/Policies

TLDR: I don't think the `topic` key was being used anyhow, so I can't foresee any unexpected consequences of removing it.

**Review by jkempster34 — APPROVED** [review]
